frontend.js

```diff
@@ -0,0 +1,287 @@
+        pathGroups: [
+          // internal
+          { pattern: "@/libs/**", group: "internal", position: "before" },
+          { pattern: "@/generated/**", group: "internal", position: "before" },
+          { pattern: "@/utils/**", group: "internal", position: "before" },
+          { pattern: "@/services/**", group: "internal", position: "before" },
+          { pattern: "@/mocks/**", group: "internal", position: "before" },
+          { pattern: "@/pageHocs/**", group: "internal", position: "before" },
+          { pattern: "@/pages/**", group: "internal", position: "before" },
+          {
+            pattern: "@/globalStates/**",
+            group: "internal",
+            position: "before",
+          },
+          { pattern: "@/models/**", group: "internal", position: "before" },
+          { pattern: "@/usecases/**", group: "internal", position: "before" },
+          {
+            pattern: "@/repositories/**",
+            group: "internal",
+            position: "before",
+          },
+          {
+            pattern: "@/components/page",
+            group: "internal",
+            position: "before",
+          },
+          {
+            pattern: "@/components/context/**",
+            group: "internal",
+            position: "before",
+          },
+          {
+            pattern: "@/components/functional/**",
+            group: "internal",
+            position: "before",
+          },
+          {
+            pattern: "@/components/model/**",
+            group: "internal",
+            position: "before",
+          },
+          {
+            pattern: "@/components/ui/**",
+            group: "internal",
+            position: "before",
+          },
+
+          // styles
+          // 最後尾にしたいのでindex扱い
+          { pattern: "@/styles/**", group: "index", position: "before" },
+          { pattern: "./**.module.css", group: "index", position: "before" },
+        ],
+      },
+    ],
+
+    /**
+     * react
+     */
+    "react/display-name": "off",
+    "react/function-component-definition": [
+      "error",
+      {
+        namedComponents: "arrow-function",
+        unnamedComponents: "arrow-function",
+      },
+    ],
+    "react/jsx-curly-newline": "off", // prettierと競合する
+    "react/jsx-filename-extension": [
+      "error",
+      {
+        extensions: [".jsx", ".tsx"],
+      },
+    ],
+    "react/jsx-indent": "off",
+    "react/jsx-key": "error",
+    "react/jsx-wrap-multilines": [
+      "error",
+      {
+        declaration: false,
+        assignment: false,
+      },
+    ],
+    "react/jsx-no-constructed-context-values": "error",
+    "react/jsx-no-useless-fragment": ["error", { allowExpressions: true }],
+    "react/jsx-one-expression-per-line": "off",
+    "react/jsx-props-no-spreading": "off",
+    "react/no-danger": "error",
+    "react/no-invalid-html-attribute": "error",
+    "react/no-unstable-nested-components": "off", // TODO: ONにすることを検討
```

Comment:
忘れてた、これここではerrorにしておいてproject側で上書きoffにしようかと思います！（そしてoff外していきたい）

```suggestion
    "react/no-unstable-nested-components": "error",
```